plugins/faucet/plugin.go

```diff
@@ -150,22 +105,12 @@ func run(plugin *node.Plugin) {
 		}
```

Comment:
Can't comment on the actual line: do we really still need `waitForMana`? After the node is bootstrapped, can't we simply check if it has mana and if not we log an error and return?